subiquity/client/client.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, opts, about_msg=None):
         self.server_updated = None
         self.restarting = False
         self.global_overlays = []
+        self.native_language = ""
```

Comment:
It looks like it will generate a major refactoring, since locale currently manages a string, and that will require adding a type. I'm all for it, but I guess that would better the done in a separate pull request.